src/webui/webapplication.cpp

```diff
@@ -486,6 +486,19 @@ Http::Response WebApplication::processRequest(const Http::Request &request, cons
     m_currentSession = nullptr;
     m_request = request;
     m_env = env;
+    m_params.clear();
+    if (m_request.method == Http::METHOD_GET) {
+        // Parse GET parameters
+        using namespace Utils::ByteArray;
+        for (const QByteArray &param : copyAsConst(splitToViews(m_request.query, "&"))) {
+            const int sepPos = param.indexOf('=');
+            if (sepPos >= 0) // ignores params without value
+                m_params[midView(param, 0, sepPos)] = midView(param, sepPos + 1);
```

Comment:
`param.mid(sepPos + 1);` ? 
m_params should hold a copy instead IMO.

... and the usage of views here is scary, why not just use QByteArray::mid() in both instance and rely on implicit sharing?

Reply:
Both m_params key and value are QString. So QByteArray::mid() will produce unnecessary data copying.

Reply:
Maybe it would be clearer to have something more obvious here? E.g.:
```c++
m_params[QString::fromUtf8(param.constData(), sepPos)] = QString::fromUtf8(param.constData() + (sepPos + 1), param.size() - (sepPos + 1));
```
or even:
```c++
const QString paramName {QString::fromUtf8(param.constData(), sepPos)};
const QString paramValue {QString::fromUtf8(param.constData() + (sepPos + 1), param.size() - (sepPos + 1))};
m_params[paramName] = paramValue;
```

Reply:
>Maybe it would be clearer to have something 

Yes! I would use the second form.

Reply:
Ok. Done.